docs/topics/mta-web-assessment-module.adoc

```diff
@@ -0,0 +1,90 @@
+
+.Prerequisites
+
+* Log into a server running {ProductShortName}
+
+.Procedure
+
+* In the {ProductShortName} user interface, select the *Migration* working mode.
+* Click *Application Inventory* in the left menu bar.
+* Click *Create new*. A *New application* dialog appears.
+* In the form, supply the following information:
+** *Name*: unique name for the new application (mandatory)
+** *Description*: short description of the application (optional)
+** *Business service*: purpose of the application (optional)
+** *Tags*: software tags that characterize the application (mandatory)
+** *Owner*: registered software owner from drop-down list (optional)
+** *Contributors*: contributors from drop-down list (optional)
+** *Comments*: relevant comments on the migrated application
+* Click *Source code* and enter the following fields.
+** *Repository type*: *Git* or *Subversion* (mandatory)
+** *Source Repository*: URL of the repository where the software code is saved (mandatory)
+** *Branch*: application code branch in the repository (mandatory)
+** *Root path*: [get info] (optional/mandatory)
+* Click *Binary* and enter the following fields.
+** *Group*: [get info] (optional/mandatory)
+** *Artifact*: [get info] (optional/mandatory)
+** *Version*: software version of the application
+** *Packaging*: [get info] (optional/mandatory)
+* Click *Create*. The dialog closes and the new application appears in the list of defined applications.
+
+[id="mta-assessment-open-app_{context}"]
+== Opening an existing application
+
+This procedure describes how to open and edit an application that has already been defined in {ProductShortName}.
+
+.Prerequisites
+
+* Log into a server running {ProductShortName}
+
+.Procedure
+
+image::mta-assessment-apps-01.png[MTA Application Inventory]
+
+. In the {ProductShortName} user interface, select the *Migration* working mode.
+. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane.
+. Click `edit` {edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.
+
+:_content-type: PROCEDURE
+[id="mta-assessment-run_{context}"]
+== Running an assessment on an application
+
+.Prerequisites
+
+* Log into a server running {ProductShortName}
+
+.Procedure
+
+* In the {ProductShortName} user interface, select the *Migration* working mode.
+* Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane.
+* Select an application from the list that appears in the main pane, click the Options menu {kebab} at the end of the row, and select *Assess* from the drop-down menu.
+
+
+image::mta-assessment-select-questionnaire-01.png[Assessment questionnaires]
+
+[start=4]
+. *Assessment Action* appears in the main pane. Select one of the questionnaires and click *Take*.
+
+[start=5]
+. *Application assessment*
+
```

Comment:
What now? Is this supposed to lead into "MTA questionnaire" (mta-introduction-questionnaire.adoc, I think)?